src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -460,7 +460,7 @@ internal bool CheckValueKind(SyntaxNode node, BoundExpression expr, BindValueKin
                     if (!isValueType || (RequiresAssignableVariable(valueKind) && (this.ContainingMemberOrLambda as MethodSymbol)?.IsEffectivelyReadOnly == true))
                     {
                         // CONSIDER: the Dev10 name has angle brackets (i.e. "<this>")
```

Comment:
While you're here, can you remove this? Clearly we aren't going to do this.

Reply:
Done